docs/podman-logs.1.md

```diff
@@ -1,9 +1,11 @@
-% podman-logs(1)
+% podman-container-logs(1)
 
 ## NAME
-podman\-logs - Fetch the logs of one or more containers
+podman\-container\-logs (podman\-logs) - Fetch the logs of one or more containers
```

Comment:
It makes me uncomfortable to have the filename be `podman-logs` but the doc string be `podman-container-logs`. Would it make sense to rename the file as long as you're making this change? And possibly, to conform with podman-info and podman-rm, keep `podman-logs` as an alternative?

In the process of reviewing this I've found a number of other problems with the man pages; I think we're going to need another CI script. (E.g. look at the NAME for podman-play.1.md and podman-generate.1.md).

Reply:
Yes we can fix that in a different PR.  I have only been fixing them as I see them. but we need someone to globally go through them all and do a rename of the docs files. And then to go and create the man page aliases.

Reply:
>  to need another CI script. 

#2648 isn't merged yet, feel free to slap a ``/hold`` on it.  I'm happy to stick in whatever else you'd like to add.  Just let me know or add in a separate PR.  We can't never has too many CI scripts :grin:

Reply:
No, it's huge. Enormous. The man pages are a mess. I'm working on a PR right now to fix the low-hanging fruit, but will need serious help for the rest. There's no way to add a CI script right now, it's too broken. Feel free to merge #2648.

Reply:
oh okay, was hoping it was maybe juss a l'il one-liner.

Suggestion: if the problems are large / wide-spread, perhaps call in a docs person to help?  No need to take it all on yourself :smiley: